ReadMe_Usage_Stancil

```diff
@@ -0,0 +1,68 @@
+ReadMe
+
+example call:
+
+./Stancel -we 4034 -he 4034 -i 100 -kv 2
```

Comment:
stencil. I suggest you do a complete search & replace in the project for
* stancil
* stancel
to stencil